devenv/start_here.sh

```diff
@@ -17,39 +12,73 @@
 export WORKON_HOME=~/.virtualenvs
 source /usr/local/bin/virtualenvwrapper.sh
 
-# mkvirtualenv b2share-evolution
-workon b2share-evolution
+export VIRTUALENV_NAME='b2share-evolution'
+export DB_NAME='b2share-evolution'
+export MACHINE_NAME=b2share
 
-# cdvirtualenv && mkdir src
+#deactivate if we're already in a virtual env
+deactivate
 
-# cdvirtualenv src
-# git clone git@github.com:EUDAT-B2SHARE/b2share.git --branch evolution
-# git clone git@github.com:EUDAT-B2SHARE/ui-frontend.git
-# cdvirtualenv src/b2share
-# pip install -r requirements.txt
-# ln -s $WORKON_HOME/b2share-evolution/src/ui-frontend $WORKON_HOME/b2share-evolution/var/invenio.base-instance/static/
+workon $VIRTUALENV_NAME
+if [ $? -ne 0 ]; then
+	echo; echo "### Make virtual env"
+	mkvirtualenv $VIRTUALENV_NAME
+	workon $VIRTUALENV_NAME
+	cdvirtualenv && mkdir src
+fi
 
-# cdvirtualenv src/ui-frontend/
-# npm install
+docker-machine start $MACHINE_NAME
+if [ $? -ne 0 ]; then
+	echo; echo "### Create docker machine for b2share"
+	docker-machine create -d virtualbox $MACHINE_NAME
```

Comment:
this step is needed for mac only. We could detect the OS and use the local docker if we are on linux.
`DOCKER_IP` would then be `127.0.0.1`